Simplify deploy to a single command (#179)
dbos-inc/dbos-docs: docs/cloud-tutorials/application-management.md

### Preliminaries

You must define [schema migrations](./database-management.md#database-schema-management) to create your application's tables.

### Deploying Applications

> You can deploy an app to run it on DBOS Cloud with a single command (run this in your application root directory):

Reviewer: ```suggestion
You can deploy an app to run it on DBOS Cloud with a single command (run this in your application's root directory):
```

```
npx dbos-cloud app deploy
```

> If you haven't provisioned a database instance (server), this command first provisions one for you. It prompts you to pick a database instance name, which is `<username>-db-server` by default.

Reviewer 2: ```suggestion
If you haven't provisioned a database instance (server), this command first prompts you to provision one.
```

For more information, see the [database management guide](../cloud-tutorials/database-management.md).

:::tip
To specify a database instance name, you can also pass it in the `-d <database-instance-name>` option.
To enable time travel for your application, specify `--enable-timetravel`.
:::

> Then, this command automatically deploys your application to under the name in its `package.json`.

Reviewer: ```suggestion
Then, this command automatically deploys your application using the `name` field in `package.json`.
```
